Sources/App/Views/PackageController/Builds/BuildIndex+Model.swift

```diff
@@ -159,10 +159,10 @@ extension BuildIndex.Model {
                     return .div(.class("failed"),
                                 .a(.href(SiteURL.builds(.value(value.id)).relativeURL()),
                                    .text("Build Failed")))
-                case let .some(value) where value.status == .pending:
+                case let .some(value) where value.status == .triggered:
                     return .div(.class("pending"), .span("Build Queued"))
                 case .some, .none:
-                    return .div(.class("unknown"), .span("Build Pending"))
+                    return .div(.class("unknown"), .span("Unknown"))
```

Comment:
To be determined what we put here exactly, both for this case and the whole section. This is where we control the cell display.

Reply:
This is what we currently have:

<img width="213" alt="CleanShot 2021-10-05 at 13 18 46@2x" src="https://user-images.githubusercontent.com/65520/136013156-47e3e3a2-32d5-408d-bc4c-054a53acd4c0.png">

with the statuses from top to bottom:

- `.ok`
- `.failed`
- `.triggered`
- `.timeout`
- `.infrastructureError`